src/App.js

```diff
@@ -1,20 +1,161 @@
-/**
- * Created by ttnd on 27/12/16.
- */
+// Create a User Registration form with fields ( FirstName, LastName, EmailId, Contact Number, Password and Confirm password).
+// Form should have a submit button. Apply form validation to incorporate following rules.
+
+
 import React from 'react'
 
-export default class App extends React.Component {
-  constructor() {
-    super()
-  }
 
+export default class App extends React.Component {
   render() {
     return (
+        <UserRegisration/>
+    );
+  }
+}
+class UserRegisration extends React.Component {
+
+  constructor(props){
+    super(props)
+    this.state = {
+      firstNameTxt : "",
```

Comment:
use more descriptive names. is `firstNameTxt` the value of the input or the validation message?